tested/languages/javascript/templates/values.js

```diff
@@ -83,10 +83,23 @@ function sendException(stream, exception) {
     if (!exception) {
         return;
     }
-    fs.writeSync(stream, JSON.stringify({
-        "message": exception.message,
-        "stacktrace": exception.stack ? exception.stack : ""
-    }));
+    if (typeof exception === "object") {
```

Comment:
Do we want to check `exception instanceof Error`, or would that be too strict? We should at least check that the message property exists; throwing random objects will still error in this implementation I think.

Reply:
> Do we want to check `exception instanceof Error`
That would be to strict, because in the solutions normal objects are thrown.

Checking the message property is a good idea.